modules/cbr-mutli-rule-module/README.md

```diff
@@ -0,0 +1,35 @@
+# Multi CBR Rule Module
+
+Creates multiple rules for Context Based Restrictions
+
```

Comment:
Can you add usage here?

Reply:
Sure I'll add it. As an FYI the other submodules do not have usage